### PR TITLE
Add missing headers to test image Dockerfiles

### PR DESCRIPTION
**Pull request by pmorie**
Adds missing header to test image Dockerfiles.

### CODE REVIEW

**Comment by rootfs** [comment]
lgtm thanks

**Comment by ixdy** [comment]
lgtm

**Comment by ixdy** [comment]
actually, cancel lgtm

File: test/images/dnsutils/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2015 Google Inc. All rights reserved.
```
Comment (ixdy) [inline test/images/dnsutils/Dockerfile]: shouldn't this be using the standard boilerplate (Kubernetes authors) instead of Google?

**Comment by pmorie** [comment]
@ixdy yes, good catch. I'll gp bacl theough and audit them all. We need verify-description to cover this too. Will do that in a follow-up.

**Comment by pmorie** [comment]
Closing this PR; I realize there are some things that need to be done to the validator, will make a new PR for those and adding headers to various files (dockerfiles, makefiles, etc)

**Pull request closed without merging**